tests/unit/conftest.py

```diff
@@ -3,19 +3,35 @@
 
 """pytest fixtures for the integration test."""
 
+# pylint: disable=protected-access
+
+import typing
+
 import ops.testing
 import pytest
 
 from charm import FlaskCharm
 
 
 @pytest.fixture(name="harness")
-def harness_fixture():
+def harness_fixture(monkeypatch) -> typing.Generator[ops.testing.Harness, None, None]:
     """Ops testing framework harness fixture."""
-    ops.testing.SIMULATE_CAN_CONNECT = True
     harness = ops.testing.Harness(FlaskCharm)
+    orig_send_signal = ops.testing._TestingPebbleClient.send_signal
 
-    yield harness
+    def patched_send_signal(self, sig, service_names):
```

Comment:
Out of curiosity, which bug?

Reply:
https://github.com/canonical/operator/pull/926